procedural/examples/banana-procedures/src/test/java/gov/nasa/ammos/aerie/procedural/examples/bananaprocedures/constraints/TestBananaConservationStub.java

```diff
@@ -0,0 +1,120 @@
+package gov.nasa.ammos.aerie.procedural.examples.bananaprocedures.constraints;
+
+import gov.nasa.ammos.aerie.procedural.constraints.Violation;
+import gov.nasa.ammos.aerie.procedural.timeline.Interval;
+import gov.nasa.ammos.aerie.procedural.timeline.collections.Instances;
+import gov.nasa.ammos.aerie.procedural.timeline.payloads.activities.AnyInstance;
+import gov.nasa.ammos.aerie.procedural.timeline.payloads.activities.Instance;
+import gov.nasa.ammos.aerie.procedural.utils.StubPlan;
+import gov.nasa.ammos.aerie.procedural.utils.StubSimulationResults;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+import gov.nasa.jpl.aerie.merlin.protocol.types.SerializedValue;
+import gov.nasa.jpl.aerie.types.ActivityInstanceId;
+import kotlin.NotImplementedError;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+public class TestBananaConservationStub {
```

Comment:
Minor: Since this and `TestBananaConservationSim` are meant to be used for user reference, could you add a class-level comment briefly going over the approach to generating simulation data. Ie, how `Stub` stubs out just the needed data, and how `Sim` runs a simulation against the built model.